Commands/Commands/queue.js

```diff
@@ -370,6 +370,9 @@ commands.registerVote = {
                     cause: 'chat_vote',
                     message: (e.message !== undefined) ? e.message : JSON.stringify(e)
                   }, e)
+                } else if (e.statusCode === 422) {
```

Comment:
I think we should leave this as it is, then refactor all our error handling and huge if statements after, changing just this one doesn't make sense when we have a mess of them, along with messy callbacks and bad promise error catching 